### ISSUE TITLE
Pause Menu save option doesn't work on the moon

### ISSUE THREAD

**Issue by Furaihai** [HarbourMasters/2ship2harkinian#773]
### Description of the bug:
Pause Menu save option isn't available after landing on the moon in the garden area

### Can you reproduce? If so please list the steps:
- [✓ ] Yes, here are the steps to reproduce the bug: 
Get to the moon, pause, try saving. The menu will simply close with no prompt to save.

### Expected behavior:
Expected a prompt to save on the moon.

### System Information:
- OS: Nobara Linux 39
- GFX Backend: OpenGL
- Game Version: Rika Charlie 1.0.2
- Rom Version: US

**Comment by garrettjoecox** [comment]
Someone can correct me if I’m wrong, but I believe this is intended behavior, I don’t recall the exact reasoning behind it

**Comment by balloondude2** [comment]
Yeah, auto/pause saving on the moon was deliberately disabled as of 1.0.1 (PR #616). 

I didn't find a clear reason why it was disabled, but looking at the issues related to the pr, some unintended state likely occurs with an auto/pause save on the moon.

**Comment by JordanLongstaff** [comment]
So if it's intended behavior, then this should be a wontfix, right?
